dbms/src/Storages/System/StorageSystemPartsColumns.cpp

```diff
@@ -25,6 +25,7 @@ StorageSystemPartsColumns::StorageSystemPartsColumns(const std::string & name)
                       {"marks_bytes_in_block", std::make_shared<DataTypeUInt64>()},
                       {"rows",                std::make_shared<DataTypeUInt64>()},
                       {"bytes",               std::make_shared<DataTypeUInt64>()},
+                      {"bytes_uncompressed",  std::make_shared<DataTypeUInt64>()},
```

Comment:
Let's rename them in both tables to `bytes_compressed` and `bytes_uncompressed`

Reply:
Ok.

Reply:
`bytes_compressed` is slightly incorrect because .mrk files are not compressed.

What about `bytes_on_disk`?

Reply:
My proposal: https://github.com/yandex/ClickHouse/pull/2052/commits/7a091243c7d1fc643a7494c04fee12f6c659185d

I like it because it is symmetric and unambiguous.